### PR TITLE
ICS-49 Volta release notes 2021-09

### PR DESCRIPTION
**Pull request by ahmedolaibrahim**
DRAFT PR FOR VOLTA Release Notes.

branches of the PR for versioning so it can contain those changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+## [0.0.2](https://github.com/energywebfoundation/iam-cache-server/pull/156) (2021-09-16)
+
+### Features
+
+- **app:** migrate from dgraph to postgres ([7be1437](https://github.com/energywebfoundation/iam-cache-server/commit/7be1437c08e75d913a49c1152058ea891766834b))
+- **app:** migrate from dgraph to postgres ([2591cf9](https://github.com/energywebfoundation/iam-cache-server/commit/2591cf9bb6bdbf4d6e5efce3367f2db4e453fa17))
```
Comment (jrhender) [inline CHANGELOG.md:10]: Are all of these new changes? I think the `volta` cache-server is already using postgres for example 🤔

Comment (ahmedolaibrahim) [reply CHANGELOG.md:10]: @jrhender this was auto generated so it generated it based on our commits but I added the Breaking Changes part myself which was why I pushed this so we can pick what's needed and not.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,127 +2,37 @@
...
-- **logging:** add loggin to files and sentry integration ([7c5a9c9](https://github.com/energywebfoundation/iam-cache-server/commit/7c5a9c96a4b61f62026c6ff85aac675fe394fb81))
-- **owner:** return subOrgs from org owner endpoint ([9a0f406](https://github.com/energywebfoundation/iam-cache-server/commit/9a0f40610343488a36bca440e0640b4ce6dcede3))
-- **owner:** return subOrgs from org owner endpoint ([9cb29d1](https://github.com/energywebfoundation/iam-cache-server/commit/9cb29d131b67dc27b53171799a75d89d84fa688e))
-- **roleDefinition:** add validation properties to fields ([33a501d](https://github.com/energywebfoundation/iam-cache-server/commit/33a501dd9f5ce28cd07037cd8ed1755bbcb69138))
-- **roleDefinition:** add validation properties to fields ([51bbf68](https://github.com/energywebfoundation/iam-cache-server/commit/51bbf682d607bc7c9144c45cd158dd16b0ee10cd))
-- **role:** get all ([e50ac18](https://github.com/energywebfoundation/iam-cache-server/commit/e50ac1801ad58d9f6e1740b329f63f991c7847fa))
+- **assets:** assets caching
+- **assets:** assets subscription events
+- **auth:** handleOriginCheck now returns unauthorized error instead of internal server error for unknown origins
```
Comment (jrhender) [inline CHANGELOG.md:11]:
```suggestion
```
This can be removed as the original change never made it to Volta cache-server

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,127 +2,37 @@
...
+- **did.controller:** remove includeClaims param for /DID endpoint
+- **did.service:** handle trailing slash in UNIVERSAL_RESOLVER_CONFIG
+- **ens:** update iam-contracts to fix ENS Sync
+- installing jwt keys
 
 ### BREAKING CHANGES
 
-- **all endpoints:** updates all endpoints to use versioning
-- **did.controller:** remove includeClaims param as not respected ([b8c7469](https://github.com/energywebfoundation/iam-cache-server/commit/b8c7469a6c42573fbc51f100dfca94613b21586c))
+- **all endpoints:** updates all endpoints to use versioning(v1)
```
Comment (jrhender) [inline CHANGELOG.md]: I would move this to the top of the file as this is the most important

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,127 +2,37 @@
...
-- **subOrgs:** not updating subOrgs definitions ([375ada4](https://github.com/energywebfoundation/iam-cache-server/commit/375ada46550ddc422dbcc03ba3eda41e7d6bd10e))
+- **assets:** add type when saving asset
+- **assets:** query accepted and created assets
+- **claim:** return claim id from claim data
+- **claim:** ignore filtering claims when roles is empty
+- **claim:** filter claims by requester
+- **claim:** issuedToken now optional
+- **did.controller:** remove includeClaims param for /DID endpoint
+- **did.service:** handle trailing slash in UNIVERSAL_RESOLVER_CONFIG
+- **ens:** update iam-contracts to fix ENS Sync
```
Comment (jrhender) [inline CHANGELOG.md:33]:
```suggestion
```
iam-contracts not yet on Volta, so this change doesn't make sense

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,127 +2,37 @@
...
-- **role.service:** check for empty conditions array ([d4b52ac](https://github.com/energywebfoundation/iam-cache-server/commit/d4b52acc62f12ee1c8a24c5421db665be2903fdc))
-- **role.service:** check for empty conditions array ([1d446d3](https://github.com/energywebfoundation/iam-cache-server/commit/1d446d3de10422ec1f660d3209d5c2876589d61c))
-- **sentry:** sentry not displaying maps ([3fe6a13](https://github.com/energywebfoundation/iam-cache-server/commit/3fe6a132c66fcc497f6d367c72eaa415d4911c34))
-- **sentry:** sentry not displaying maps ([9c695e4](https://github.com/energywebfoundation/iam-cache-server/commit/9c695e41fe07902b71c07675f6325267a8565e95))
-- **subOrgs:** not updating subOrgs definitions ([ee70734](https://github.com/energywebfoundation/iam-cache-server/commit/ee707344b4e85e0730692a4cf2a4c166c5a3b88a))
-- **subOrgs:** not updating subOrgs definitions ([375ada4](https://github.com/energywebfoundation/iam-cache-server/commit/375ada46550ddc422dbcc03ba3eda41e7d6bd10e))
+- **assets:** add type when saving asset
+- **assets:** query accepted and created assets
```
Comment (jrhender) [inline CHANGELOG.md:26]:
```suggestion
```
We don't have assets on volta yet, so can remove these

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+### Bug Fixes
+
+- **claim:** return claim id from claim data
+- **claim:** ignore filtering claims when roles is empty
+- **claim:** filter claims by requester
+- **claim:** issuedToken now optional
+- **did.controller:** remove includeClaims param for /DID endpoint
+- **did.service:** handle trailing slash in UNIVERSAL_RESOLVER_CONFIG
+- installing jwt keys
```
Comment (jrhender) [inline CHANGELOG.md:32]: What is "installing jwt keys"?

Comment (ahmedolaibrahim) [reply CHANGELOG.md:32]: @jrhender  it was a fix [here](https://github.com/energywebfoundation/iam-cache-server/pull/156/commits/d5f81dac8f30d7c7071d11a95c2160aa43c5af1e)

**Comment by jrhender** [comment]
Looks good. Please also add some instructions to the cache-server readme on how you see standard-version being used in the furture.

**Review by jrhender — APPROVED** [review]
Looks good, thanks!

**Pull request merged**